package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A workflow for Angular made with Webpack",
   "scripts": {
     "build": "rimraf dist && webpack --bail --progress --profile",
-    "server": "webpack-dev-server --host 0.0.0.0 --port 8080 --history-api-fallback --inline --progress",
+    "server": "webpack-dev-server --history-api-fallback --inline --progress",
```

Comment:
why was the command changed?

Reply:
I reverted it by this commit. In the first place, I wanted to use webpack-dev-server and I needed to change host to 0.0.0.0 to make it work (to fix ERR_CONNECTION_REFUSED error). But then I realized that it's better to use separated web-server (in this case express) and build command because webpack-dev-server is not good for production.